images/c/Dockerfile.standalone

```diff
@@ -1,9 +1,5 @@
 FROM whanos-c:latest
```

Comment:
A two step dockerfile should be used for this. We don't need gcc/make or source files on the standalone.

It should probably be something like this:

```Dockerfile
FROM whanos-c:latest as builder
COPY . /app
WORKDIR /app
RUN make

FROM alpine
SHELL ["/usr/bin/bash", "-c"]
CMD ["./compiled-app"]
```

PS: I did not test the dockerfile.

Reply:
alpine doesn't have bash

Reply:
FROM bash:latest then

Reply:
It will be installed to /usr/local/bin/bash if you use this image.

Reply:
i will keep this in mind